gradle.properties

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-ideaVersion = IC-15.0.6
+ideaVersion = IU-162.1447.7
```

Comment:
And when testing with IJ < 2016.2 locally, the tests will fail. I guess there's no way around this without maintaining multiple versions of our plugin (which seems like a more painful thing to do for now).